### PR TITLE
VCDL-252 taxonomy changes

### PR DESCRIPTION
**Pull request by jennamwong**
DO NOT MERGE!!!!!!
Taking out instances of Terraform Cloud and replacing with HCP Terraform.
Cannot merge until April 22, 2024.

### CODE REVIEW

File: instruqt-tracks/terraform-cloud-aws/track.yml [hunk 1]
```diff
@@ -21,18 +18,24 @@ description: |-
...
+idle_timeout: 300
+timelimit: 14400
 lab_config:
   overlay: false
   width: 25
   position: right
+  sidebar_enabled: true
+  feedback_recap_enabled: true
+  loadingMessages: true
+checksum: "16868046364266727064"
```
Comment (syedQuadri7) [inline instruqt-tracks/terraform-cloud-aws/track.yml:41]:
```suggestion

```

File: instruqt-tracks/terraform-cloud-aws/track.yml [hunk 2]
```diff
@@ -1,16 +1,13 @@
-# Copyright (c) HashiCorp, Inc.
-# SPDX-License-Identifier: MPL-2.0
-
-slug: terraform-cloud-aws
-id: 2hgi8e9hftli
-title: Terraform Cloud with AWS
+slug: wip-vcdl-249-hcp-terraform-aws
+id: jamndfnxfdi1
```
Comment (syedQuadri7) [inline instruqt-tracks/terraform-cloud-aws/track.yml:2]:
```suggestion
slug: hcp-terraform-aws
```

File: instruqt-tracks/terraform-cloud-aws/track.yml [hunk 3]
```diff
@@ -21,18 +18,24 @@ description: |-
...
 icon: https://storage.googleapis.com/instruqt-hashicorp-tracks/logo/terraform.png
 tags:
-- terraform
-- aws
 - cloud
 - enterprise
-owner: hashicorp
+- aws
+- terraform
+owner: hashicorp-field-ops
```
Comment (syedQuadri7) [inline instruqt-tracks/terraform-cloud-aws/track.yml:25]:
```suggestion
owner: hashicorp
```

File: instruqt-tracks/terraform-cloud-gcp/track.yml [hunk 4]
```diff
@@ -1,16 +1,16 @@
 # Copyright (c) HashiCorp, Inc.
 # SPDX-License-Identifier: MPL-2.0
 
-slug: terraform-cloud-gcp
+slug: hcp-terraform-gcp
 id: mjfywxvlky8d
```
Comment (syedQuadri7) [inline instruqt-tracks/terraform-cloud-gcp/track.yml:5]:
```suggestion

```

File: instruqt-tracks/terraform-cloud-aws/track.yml [hunk 5]
```diff
@@ -1,16 +1,12 @@
-# Copyright (c) HashiCorp, Inc.
-# SPDX-License-Identifier: MPL-2.0
-
-slug: terraform-cloud-aws
-id: 2hgi8e9hftli
-title: Terraform Cloud with AWS
+slug: hcp-terraform-aws
```
Comment (syedQuadri7) [inline instruqt-tracks/terraform-cloud-aws/track.yml:1]:
```suggestion
# Copyright (c) HashiCorp, Inc.
# SPDX-License-Identifier: MPL-2.0

slug: hcp-terraform-aws
```

**Review by syedQuadri7 — CHANGES_REQUESTED** [review]
needs to add copyright

File: instruqt-tracks/terraform-cloud-aws/track.yml
Comment (syedQuadri7) [inline instruqt-tracks/terraform-cloud-aws/track.yml:1]: need to add 
```
# Copyright (c) HashiCorp, Inc.
# SPDX-License-Identifier: MPL-2.0
```

**Comment by jennamwong** [comment]
Combined this PR into VCDL-251 branch

**Pull request closed without merging**